packages/gatsby-theme-carbon/src/components/Code/Code.js

```diff
@@ -14,9 +14,18 @@ import Sidebar from './Sidebar';
 
 import useMetadata from '../../util/hooks/useMetadata';
 
-const Code = ({ children, className: classNameProp = '', path, src }) => {
-  const [hasMoreThanNineLines, setHasMoreThanNineLines] = useState(false);
-  const [shouldShowMore, setShouldShowMore] = useState(false);
+const Code = ({
+  children,
+  className: classNameProp = '',
+  overflow = 9,
+  path,
+  src,
+}) => {
+  const isOverFlowEnabled = overflow > 0;
+  const shouldOverflow = children.split('\n').length > overflow;
```

Comment:
I moved this out of `getLines` to fix this warning:
![image](https://user-images.githubusercontent.com/5770007/102811067-cdaec000-4392-11eb-8799-1579845106e2.png)